tests/test_install_mender_sh.py

```diff
@@ -337,6 +337,7 @@ def test_upgrade_mender_v4_series_explicit_auth_update(
         check_installed(generic_debian_container, "mender-configure")
 
 
+@pytest.mark.skip("skip until Mender 4.0.0 is properly released")
```

Comment:
The purpose of these tests were to verify the *upgrade* path from faulty v4 to epoch `1:` v3 with different variations. The upcoming release will publish different packages and these test won't run (as originally intended) with the new v4 packages.

For now, great to disable them and progress with the review.

However, for later, I think the best would be to save these packages *for tests purposes* somewhere in this repository and modify the tests, so that we keep testing in regression the upgrade to v3 path.

Note that we have now a [new set of tests](https://github.com/mendersoftware/mender-dist-packages/blob/650800c6b3495c323b98ed96f47a2c9f581dfd8d/tests/test_install_mender_sh.py#L228-L337) for the `mender-client4` package and it's relation with v3 `mender-client`.

Reply:
> (...) However, for later, I think the best would be to save these packages _for tests purposes_  (...)

I mean rescue them from a backup into this repo or some other test location

Reply:
Reworked the PR to download the old files from a special location.